backend/tests/src/tests/json-schemas/auth/sign-up/sign-up-response.ts

```diff
@@ -0,0 +1 @@
+export { signUpErrorResponseSchema } from '~/tests/json-schemas/shared/error/sign-up-error-response.schema';
```

Comment:
Don't export it here. Import if from shared.
Also, you even don't need the `json-schemas/auth` folder. Remove it